backend/src/main/java/de/neuefische/backend/model/Appointments.java

```diff
@@ -0,0 +1,35 @@
+package de.neuefische.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document("appointments")
+public class Appointments {
+
+    @Id
+    private String id;
+    private String username;
+    private String appointmentName;
+    private LocalDateTime creationDate;
+    private Date endDate;
+
+    public static Appointments newAppointment (String username, String appointmentName, LocalDateTime creationDate, Date endDate){
```

Comment:
Why write a factory method when you have a builder?

Reply:
If I remember correctly, we learned it like that a few weeks ago. Since the i've been using these factory methods.
Would it just be shorter if I just the builder in the service or am i completely misunderstanding something?

Reply:
I would go with the builder as it is more expressive than a factory method - for the factory method you have to remember the order of arguments.